r/inst/build_arrow_static.sh

```diff
@@ -81,10 +96,14 @@ ${CMAKE} -DARROW_BOOST_USE_SHARED=OFF \
     -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON \
     -DCMAKE_UNITY_BUILD=${CMAKE_UNITY_BUILD:-OFF} \
     -Dxsimd_SOURCE=${xsimd_SOURCE:-} \
+    -DARROW_USE_SCCACHE=ON \
     ${EXTRA_CMAKE_FLAGS} \
     -G ${CMAKE_GENERATOR:-"Unix Makefiles"} \
     ${SOURCE_DIR}
 
 ${CMAKE} --build . --target install
 
+echo "=== sccache stats after the build ==="
+sccache --show-stats
```

Comment:
Could you show a build log that we can see this?

Reply:
https://github.com/ursacomputing/crossbow/runs/7257889831?check_suite_focus=true#step:6:2578

The other Linux builds of that matrix and the wincpp job also displays them if you want to compare :D